docs/get-started/getting-started.md

```diff
@@ -375,9 +375,97 @@ and `GpuColumnarExchange`.  Those correspond to operations that run on the GPU.
 
 ![Join Example on Spark SQL UI](../img/join-sql-ui-example.png)
 
+## Enabling RapidsShuffleManager
+---
+**NOTE**
+
+The _RapidsShuffleManager_ is a beta feature!
+
+---
+
+The _RapidsShuffleManager_ is an implementation of the `ShuffleManager` interface in Apache Spark,
+that allows custom mechanisms to exchange shuffle data. The _RapidsShuffleManager_ has two components:
+a spillable cache, and a transport that can utilize _Remote Direct Memory Access (RDMA)_ and high-bandwidth 
+transfers within a node that has multiple GPUs. This is possible because the plugin utilizes 
+[Unified Communication X (UCX)](https://www.openucx.org/) as its transport.
+
+- **Spillable cache**: This store keeps GPU data close by where it was produced, and spills to host memory
+or disk when running out of space due to limited GPU memory, or configuration (when going from host
+to disk). Local tasks (to the producing executor) will short-circuit read from this cache.
+
+- **Transport**: Handles block transfers between executors using various means like: _NVLink_, _PCIe_, _Infiniband (IB)_, 
+_RDMA over Converged Ethernet (RoCE)_ or _TCP_, and as configured in UCX, in these scenarios:
+  - _GPU-to-GPU_: Shuffle blocks that were able to fit in GPU memory.
+  - _Host-to-GPU_ and _Disk-to-GPU_: Shuffle blocks that spilled to host (or disk), but will be manifested 
+  in the GPU in the downstream Spark task.
+
+In order to enable the _RapidsShuffleManager_, please follow these steps:
+
+1) (If you don't have Mellanox hardware go to *step 2*) If you have Mellanox NICs and an Infiniband(IB)
+or RoCE network, please ensure you have the MLNX_OFED [driver installed](https://www.mellanox.com/products/infiniband-drivers/linux/mlnx_ofed), 
+and the [`nv_peer_mem` kernel module](https://www.mellanox.com/products/GPUDirect-RDMA). This will 
+need to be installed on every machine for RDMA transfers to work between the GPU and the NIC. 
+
+Without `nv_peer_mem`, IB/RoCE-based transfers will not be zero-copy from the GPU.
```

Comment:
```suggestion
In order to enable the _RapidsShuffleManager_, please follow these steps (If you don't have Mellanox hardware go to *step 2*):

1) If you have Mellanox NICs and an Infiniband(IB) or RoCE network, please ensure you have the MLNX_OFED [driver installed](https://www.mellanox.com/products/infiniband-drivers/linux/mlnx_ofed), 
and the [`nv_peer_mem` kernel module](https://www.mellanox.com/products/GPUDirect-RDMA). This will 
need to be installed on every machine for RDMA transfers to work between the GPU and the NIC. 

With `nv_peer_mem`, IB/RoCE-based transfers will be zero-copy from the GPU.
```

Reply:
Thanks @sameerz I phrased it slightly differently in https://github.com/NVIDIA/spark-rapids/pull/390/commits/1d9d46eb5bba4ae4351790f81be80093b02fe636. For the `nv_peer_mem` case, UCX gets the _choice_ to use RDMA, but it's not a requirement, as it can decide that staging through the host is lower cost in some scenarios.